src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -1377,6 +1379,12 @@ well as menu structures (for main menu and popup menus).
         menuLabel="Zoom VCS"
         windowMode="main"/>
         
+   <cmd id="layoutZoomTutorial"
```

Comment:
Also think about having a command on the View menu for moving focus to the Tutorial pane. Important for accessibility, both screen reader users and for keyboard-only users.

This area is a bit of a mess, see #5775 for more considerations (nothing you need to do about it at the moment!)